src/reference/90-Developers-Guide/06-Datatype/00.md

```diff
@@ -5,12 +5,12 @@ out: Datatype.html
 Datatype
 --------
 
-sbt-datatype is a code generation library and an sbt autoplugin that generates
+`sbt-datatype` is a code generation library and an sbt autoplugin that generates
```

Comment:
Stylistic choice, my habit but I don't feel strongly either way.

One could raise similar bikeshedding over the use of "sbt" throughout the docs, that it's all-lowercase, whether it's ever capitalized at the beginning of a sentence (I say no, it's being treated like a brand mark), etc.

I don't particularly care either way, but what I would ask is that if you *do* feel strongly about it, please define a documentation style guide somewhere :smile: